pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -620,3 +626,44 @@ func DeleteIPsecEncryptRoute() {
 		}
 	}
 }
+
+func keyfileWatcher(ctx context.Context, watcher *fswatcher.Watcher, keyfilePath string, nodediscovery *nodediscovery.NodeDiscovery) {
+	for {
+		select {
+		case event := <-watcher.Events:
+			if event.Name != keyfilePath {
+				continue
+			}
+
+			log.WithField(logfields.Path, keyfilePath).Debugf("Received %s fsnotify event for IPSec keyfile", event.Op.String())
```

Comment:
Should we make this an Info message? I'm not sure if we have other log messages for when the secret changes.